Directory.Build.props

```
@@ -3,7 +3,7 @@
 
     <PropertyGroup>
         <Nullable>enable</Nullable>
-        <Version>1.11.0</Version>
+        <Version>1.12.0-alpha1</Version>
```

```suggestion
        <Version>1.11.0</Version>
```
This is a really low risk change, so I don't think we need a pre-release version. I can just release v1.12.0 after this merges.